Sub/Src/ComputerVision/Nodes/CamNode.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import cv2
+import math
```

Comment:
Please also remove all the darknet code (except the weights) from the repo...we will just download the repo on the Tegra.

Reply:
The repo isn't on the git though. I can get rid of the extra weights and name files.